VRTweaks/SnapTurn/SnapTurning.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace VRTweaks.SnapTurn
 {
-    [HarmonyPatch(typeof(MainCameraControl), nameof(MainCameraControl.Update))]
```

Comment:
This wouldn't compile for me because Update is a private method in MainCameraControl. I set it to a string here since I didn't know any other way